discuss/customizations/call_notes_main_functionality.rst

```diff
@@ -0,0 +1,88 @@
+================
+Note Activitiy
+================
+
+New activity type called *Note* added next to currently 
+available generic *Other* type and *Meeting* (that allows 
+to create meeting from activity).
+
+If you select *Note* category on your activity type (most likely 
+on Call activity type), it will behave as a goal activity, where you 
+will need to select goal and contact. 
+
+Configuration
+=============
+
+To start using this new activity category, module :code:`mail_activity_note`
+must be installed and *Note* activity type added.
+
+To add this new activity type, go to :menuselection:`CRM --> Configuration --> Activity Types`
+and create  new activity type with category *Note*:
+
+.. image:: media/note_activity_category.png
+    :align: center
+
+Contact can be made mandatory on activity type, by checking *Contact required* option. 
+If not checked, it will be optional.
+
+*Hide popover* is alvays mandatory. This part must depend on 
+module :code:`mail_activity_popover_toggle` that allows to hide 
```

Comment:
You could just add a link to it's documentation like this: ```See more :doc:`mail_activity_popover_toggle`.``` (not sure if path is correct, need to check).

Reply:
You mean instead of :code:`mail_activity_popover_toggle`, I shoud write :doc:`mail_activity_popover_toggle`?